license.md

```diff
@@ -1,23 +1,7 @@
-Copyright (c) 2015, SilverStripe
-All rights reserved.
+Copyright (c) 2015 Christopher Pitt
```

Comment:
No I mean, the original being you. The "old" module is a separate module. It just happens to be put in the same repo as this one. ;)